server/datastore/psql/database.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import(
+	"fmt"
+	"database/sql"
+	//"github.com/jackc/pgx/v4"
+	_ "github.com/lib/pq"
+)
+
+
+const (
+	host = "localhost"
+	port = 5432
+	user = "postgres"
+	password = "summer2020"
+	dbname = "nsl_dev"
```

Comment:
shouldn't have these hardcoded variables. Add it to the `server/config.yaml` file